lib/pbench/server/database/models/users.py

```diff
@@ -143,14 +182,17 @@ def is_admin(self) -> bool:
         Returns:
             True if the user's role is ADMIN, False otherwise.
         """
-        return self.role is Roles.ADMIN
+        return Roles.ADMIN.name in self.roles
 
-    @staticmethod
-    def is_admin_username(username: str) -> bool:
-        """Determine if the given user name is an admin user.
+    def as_json(self) -> JSONOBJECT:
+        """Return a JSON object for this User object.
 
         Returns:
-            True if the user is an admin; False otherwise.
+            A JSONOBJECT with all the object fields mapped to appropriate names.
         """
-        admins = ["admin"]
-        return username in admins
+        return {
+            "id": self.id,
+            "oidc_id": self.oidc_id,
+            "username": self.username,
+            "roles": self.roles,
+        }
```

Comment:
Why do we have both this function and `get_json()`?  (I'm thinking that at least one of the two should probably be given a more descriptive name....)